<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGeneratePassword</class>
 <widget class="QDialog" name="DialogGeneratePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>密码生成器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelCharTypes">
     <property name="text">
      <string>字符:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBoxLow">
     <property name="text">
      <string>小写(&amp;L)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QSpinBox" name="spinBoxUpperSize">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="labelUpperSize">
     <property name="text">
      <string>位</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QCheckBox" name="checkBoxUpper">
     <property name="text">
      <string>大写(&amp;U)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QSpinBox" name="spinBoxLowerSize">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="labelLowerSize">
     <property name="text">
      <string>位</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBoxNumber">
     <property name="text">
      <string>数字(&amp;N)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QSpinBox" name="spinBoxNumberSize">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="labelNumberSize">
     <property name="text">
      <string>位</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="checkBoxSpecial">
     <property name="text">
      <string>特殊(&amp;S)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QSpinBox" name="spinBoxSpecialSize">
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="labelSpecialSize">
     <property name="text">
      <string>位</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPasswordLen">
     <property name="text">
      <string>长度:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBoxPassLen">
     <property name="value">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelPasswd">
     <property name="text">
      <string>密码:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="5">
    <widget class="QLineEdit" name="lineEditPasswd"/>
   </item>
   <item row="3" column="0" colspan="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>292</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>生成(&amp;G)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6" colspan="2">
    <widget class="QPushButton" name="pushButtonCopy">
     <property name="text">
      <string>复制(&amp;C)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
